app/styles/modules/_custom-rows.scss

```diff
@@ -87,7 +86,15 @@ input[type='checkbox'] {
   cursor: pointer;
   display: inline-block;
   height: 18px;
-  margin-inline-start: 4px;
+
```

Comment:
@vbudhram - should we get this uplifted to train-122, it's in prod now. :(

Reply:
@shane-tomlinson I don't think it is super urgent? but I don't mind switching to 122 if you like.

Reply:
@vbudhram - it doesn't hurt functionality, it's just ugly.